apps/explorer/src/pages/object-result/ObjectResult.tsx

```diff
@@ -8,7 +8,7 @@ import { ErrorBoundary, PageLayout } from '~/components';
 import { PageHeader } from '~/components/ui';
 import { ObjectView } from '~/pages/object-result/views/ObjectView';
 import { translate, type DataType } from './ObjectResultType';
-import PkgView from './views/PkgView';
+import { PkgView } from './views/PkgView';
 import { TokenView } from './views/TokenView';
```

Comment:
can you add an index.ts file to organize these imports? that way, we can simplify them like this:
```suggestion
import { PkgView, TokenView } from './views';
```